src/lib/core/datetime.c

```diff
@@ -58,15 +74,6 @@ tnt_datetime_now(struct datetime *now)
 	now->tzoffset = tm.tm_gmtoff / 60;
 }
 
-size_t
-tnt_datetime_strftime(const struct datetime *date, char *buf, size_t len,
-		      const char *fmt)
-{
-	struct tm tm;
-	datetime_to_tm(date, &tm);
-	return strftime(buf, len, fmt, &tm);
```

Comment:
Why did you move this function above? It would look cleaner in the diff if you would update it in-place. The whole `tnt_datetime_strftime` I mean.

Reply:
Hmm, yes, I've normalized underlying patch, to make patch smaller.